documentation/en/actor-events-api.md

```diff
@@ -0,0 +1,434 @@
+# Actor Events and Lotus APIs
+
+* [Background](#background)
+* [ActorEvent structure](#actorevent-structure)
+* [Querying Lotus for ActorEvents](#querying-lotus-for-actorevents)
+* [Retrieving events from message receipts](#retrieving-events-from-message-receipts)
+* [Current builtin actor event schemas](#current-builtin-actor-event-schemas)
+  * [Verified registry actor events](#verified-registry-actor-events)
+    * [Verifier balance](#verifier-balance)
+    * [Allocation](#allocation)
+    * [Allocation removed](#allocation-removed)
+    * [Claim](#claim)
+    * [Claim updated](#claim-updated)
+    * [Claim removed](#claim-removed)
+  * [Market actor events](#market-actor-events)
+    * [Deal published](#deal-published)
+    * [Deal activated](#deal-activated)
+    * [Deal terminated](#deal-terminated)
+    * [Deal completed](#deal-completed)
+  * [Miner actor events](#miner-actor-events)
+    * [Sector precommitted](#sector-precommitted)
+    * [Sector activated](#sector-activated)
+    * [Sector updated](#sector-updated)
+    * [Sector terminated](#sector-terminated)
+
+## Background
+
+Actor events are a fire-and-forget mechanism for actors in Filecoin to signal events that occur during execution of their methods to external observers. The events themselves are not stored on chain, but are emitted by the actor and may be stored in a node's event store. They are intended to be used by tooling and applications that need to observe and react to events that occur within the chain.
```

Comment:
```suggestion
Actor events are a fire-and-forget mechanism for actors in Filecoin to signal events that occur during execution of their methods to external observers. The events themselves are not stored on chain (in state), but are emitted by the actor and may be stored in a node's event store. They are intended to be used by tooling and applications that need to observe and react to events that occur within the chain.
```

Reply:
bit more complicated than that, I've adjusted this para to read:

Actor events are a fire-and-forget mechanism for actors in Filecoin to signal events that occur during execution of their methods to external observers. Actor events are intended to be used by tooling and applications that need to observe and react to events that occur within the chain. The events themselves are not stored in chain state, although a root CID for an array (AMT) of all events emitted for a single message is recorded on message receipts, which are themselves referenced as an array (AMT) in the `ParentMessageReceipts` in each `BlockHeader` of a tipset. A node may optionally retain historical events for querying, but this is not guaranteed and not essential as it does not affect the chain state.